float32 x
float32 y
float32 w
float32 h
string bbox_class
float32 confidence


